docs/DeveloperGuide.md

```diff
@@ -147,3 +148,40 @@ Manage contacts and schedule of students faster than a typical mouse/GUI driven
     - 3b1. Teacher’s Pet shows a message indicating student’s attendance is already marked as present.
 
         Use case resumes at step 2.
+
+### Use case: Allocate a slot for future class
+
+**MSS**
+
+1. Teacher requests to find the next available slot for class
+2. Teacher discusses with the student about whether the proposed slot is possible
+3. Teacher [update](#use-case-update-a-student-contact-detail) the student record with the next class date
```

Comment:
I feel that MSS should be alternating between the Teacher and the system? Correct me if I'm wrong.
ie:
```
1. Teacher requests to find the next available slot for class
2. Teacher's Pet shows the next available time slot.
3. Teacher discusses with the student about whether the proposed slot is possible
4. Teacher [update](#use-case-update-a-student-contact-detail) the student record with the next class date
```